app/views/dashboards/show.html.erb

```diff
@@ -10,7 +10,7 @@
     </section>
 
     <% if @presenter.approvals.any? %>
-      <section class="mb-5">
+      <section class="mb-5" id="approvals">
```

Comment:
Makes for a more targeted feature spec.

Reply:
consider https://github.com/sul-dlss/happy-heron/pull/607/files/ea86cbb9bf8bb5743dc89bea77341db965f78ab8#diff-171048dc70a2f5431a6ecc157b9b17eb786dc5bd1710c795c2c69551d7ac9a63 instead.

Reply:
Nice, I dig it.